app/src/features/apiClient/screens/apiClient/components/clientView/components/Collection/CollectionView.tsx

```diff
@@ -75,9 +75,12 @@ export const CollectionView = () => {
           auth: newAuthOptions,
         },
       };
-      return upsertApiRecord(user.details?.profile?.uid, record, teamId).then((result) => {
-        onSaveRecord(result.data);
-      });
+      return upsertApiRecord(user.details?.profile?.uid, record, teamId)
+        .then((result) => {
+          // fix-me: to verify new change are broadcasted to child entries that are open in tabs
```

Comment:
is this done?

Reply:
As discussed, and was evident in the finally released [base PR](https://github.com/requestly/requestly/pull/2409), rendering correctness is no longer a concern and hence this comment is meant to be addressed only when we decide to fix that. So no, not done right now.